src/styles/Progress.module.css

```diff
@@ -0,0 +1,51 @@
+.progressCont {
+  display: flex;
+  align-items: center;
+  gap: 10px;
+}
+
+.progress {
+  text-align: center;
+}
+
+.progress :nth-child(1) {
+  font-size: 1.5rem;
+  font-weight: bold;
+}
+
+.progress :nth-child(2) {
+  font-size: smaller;
+  font-weight: lighter;
+}
+
+.CircularProgress {
+  width: 100%;
+  max-width: 100px;
+}
+
+@media screen and (max-width: 768px) {
+  .progressCont {
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 10px;
+  }
+
+  .CircularProgress {
+    width: 100%;
+    max-width: 10vmin;
+  }
+
+  .progress :nth-child(1) {
+    font-size: 1rem;
+    font-weight: bold;
+  }
```

Comment:
- [ ] Please be reminded that this activity is focused on styling, we want the design of your app to closely mimics the [design template](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) on zeplin. 

**Your design:**
![image](https://github.com/christianonoh/book-town/assets/12745474/e21947d7-fd8b-49f9-a3e0-a3ed66a15abb)

**The original design:**
![image](https://github.com/christianonoh/book-town/assets/12745474/67519f48-79c1-4968-8afa-a2d89a5c9451)